msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties

```diff
@@ -415,6 +417,8 @@ perm_choose_instruction_topic=This will be the default when grading in this topi
 perm_choose_instruction_more_link=(More?)
 perm_choose_instruction_forum_more=You can override this association in the child topics by changing it in the Topic settings.
 perm_choose_instruction_topic_more=You can assign any entry from the Gradebook to contributions in this topic. This is just setting the default.
+perm_create_task_forum=Create task to students on their dashboard tasks widget for this forum.
+perm_create_task_topic=Create task to students on their dashboard tasks widget for this topic.
```

Comment:
```suggestion
perm_create_task_forum=Create forum task in Dashboard tasks widget.
perm_create_task_topic=Create topic task in Dashboard tasks widget.
```

Reply:
Thanks, I've changed it as you suggested